src/call/mod.rs

```diff
@@ -45,31 +45,63 @@ pub struct GrpcByteBuffer {
     pub raw: *mut grpc_sys::GrpcByteBuffer,
 }
 
-impl Default for GrpcByteBuffer {
-    fn default() -> Self {
+impl GrpcByteBuffer {
+    pub fn push(&mut self, mut slice: GrpcSlice) {
+        unsafe {
+            grpc_sys::grpcwrap_byte_buffer_add(self.raw as _, &mut slice as _);
+            grpc_sys::grpcwrap_slice_drop_without_unref(slice);
```

Comment:
You can avoid an empty `grpcwrap_slice_drop_without_unref` by transferring the ownership of `GrpcSlice` to `grpcwrap_byte_buffer_add`.